hoot-core/src/main/cpp/hoot/core/util/HootException.cpp

```diff
@@ -22,7 +22,7 @@
  * This will properly maintain the copyright information. DigitalGlobe
  * copyrights will be updated automatically.
  *
- * @copyright Copyright (C) 2015, 2016, 2017 DigitalGlobe (http://www.digitalglobe.com/)
+ * @copyright Copyright (C) 2015, 2016, 2017, 2018 DigitalGlobe (http://www.digitalglobe.com/)
```

Comment:
Here I made a change in the file, that change caused unit tests to fail, so I backed it out.  But in doing so, `git` already saw my change and wants the copyright info updated.  Whatever.